docs/Documentation/Applications/vasp.md

```diff
@@ -0,0 +1,537 @@
+
+```
+I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 16 vasp_std
+```
+If your multi-node Open MPI VASP job is crashing on Vermilion, replace a call to load an openmpi module with the following lines. The OMPI_MCA_param variable sets ens7 as the interconnect. 
+
+```
+module use /nopt/nrel/apps/220525b/level01/modules/lmod/linux-rocky8-x86_64/gcc/12.1.0
+module load openmpi
+OMPI_MCA_param="btl_tcp_if_include ens7"
+```
+
+## Troubleshooting
+Known issues: 
+
+- VASP on Vermilion can crash on multi-node jobs (see [Workarounds on Vermilion section](vasp.md#workarounds)) 
+
+- OpenMPI more stable for multi-node jobs
+
+- OpenACC build eats up more memory but is faster (Cuda build doesn't eat up as much memory)
+
+- VASP 6.1.1 crashes for HSE calculations (due to specific version of compiler and vasp) - use a newer version instead
+
+
+## Performance Recommendations
+Studies have been carried out to evaluate the performance of VASP on Swift and Eagle using NREL's benchmarks. The inputs files for the two benchmarks are located in NREL's benchmark repository for VASP: [ESIF VASP Benchmarks](https://github.com/NREL/ESIFHPC3/tree/master/VASP). 
+
+Benchmark 1 is a system of 16 atoms (Cu<sub>4</sub>In<sub>4</sub>Se<sub>8</sub>) and represents a typical high-accuracy band structure calculation, involving bootstrapping from an initial approximate GGA wavefunction, through a hybrid HSE function, to a final band structure from a GW calculation. Benchmark 2 is a system of 519 atoms (Ag<sub>504</sub>C<sub>4</sub>H<sub>10</sub>S<sub>1</sub>) and represents a typical surface catalysis study calculation, with large unit cell and k-point sampling limited to the Gamma point. A single model chemistry (DFT functional and plane-wave basis) is employed, and strong scaling with respect to MPI rank count is of interest. The input files can be accessed in the [bench1](https://github.com/NREL/ESIFHPC3/tree/master/VASP/bench1/input) and [bench2](https://github.com/NREL/ESIFHPC3/tree/master/VASP/bench2/input) folders.
+
+
+Below, a few performance recommendations around resource requests and MPI are listed for each cluster.
+
+??? example "Eagle"
+    Intel MPI is recommended over Open MPI. Using an Intel MPI build of VASP and running over Intel MPI, Benchmark 2 ran in an average of 50% of the time as the same calculations using an Open MPI build of VASP over Open MPI. For Benchmark 1, Intel MPI calculations ran in an average of 63.5% of the time as Open MPI calculcations. 
+
+??? example "Swift"
+    Intel MPI is recommended over Open MPI for all VASP calculations on Swift. Using an Intel MPI build of VASP and running over Intel MPI, Benchmark 2 ran in average of 76%, 72% and 46% of the time as the same calculations using an Open MPI build of VASP over Open MPI on 32, 64 and 128 CPUs/node, respectively. For Benchmark 1, Intel MPI calculations ran in an average of 76.89% of the time as Open MPI calculcations. 
```

Comment:
This paragraph is huge, I think it would help to identify the key points, and shorten paragraph or make into bullet points.

Reply:
I do want to make the all vasp documentation more concise, I was thinking of taking care of this when I add in the Kestrel documentation and make my own pull request, what do you think?

Reply:
Sounds good. Created issue #405 to track needed improvement.